migrations/versions/60_archive_current_services.py

```diff
@@ -8,7 +8,7 @@
 
 # revision identifiers, used by Alembic.
 revision = '60_archive_current_services'
-down_revision = '50_remove_updated_details'
+down_revision = '60_add_acknowledged_column'
```

Comment:
Updating the down revision of previous migrations sounds like a bad idea. Surely this will fail if the previous migration has already gone out?